client/src/Components/TaskForm/TaskForm.js

```diff
@@ -0,0 +1,134 @@
+// Styling
+// import "./Task.css";
+import useInput from "../../hooks/useInput";
+import { useState } from "react";
+
+
+// Add verifications here
+const isNotEmpty = (value) => value.trim() !== "";
+// const isEmail = (value) => value.includes("@");
+
+// Card for general styling from UI
+import Card from "../UI/Card/Card";
+import DateSelector from "./DateSelector";
+import DropdownButton from "react-bootstrap/DropdownButton";
+import DropdownItem from "react-bootstrap/esm/DropdownItem";
+
+const TaskForm = ({ onSubmit }) => {
+
+    const {
+			value: taskTitleValue,
+			isValid: taskTitleIsValid,
+			hasError: taskTitleHasError,
+			valueChangeHandler: taskTitleChangeHandler,
+			inputBlurHandler: taskTitleBlurHandler,
+			reset: resetTaskTitle,
+		} = useInput(isNotEmpty);
+
+		let formIsValid = false;
+
+		if (taskTitleIsValid) {
+			formIsValid = true;
+		}
+
+		const submitHandler = (event) => {
+			event.preventDefault();
+
+			if (!formIsValid) {
+				return;
+			}
+			//need to post the task details
+			console.log("Submitted");
+			console.log(taskTitleValue);
```

Comment:
...and the current handler doesn't do anything with the data.

Reply:
This here too will be modified (original plan that we can change if there is any suggested alternative) to include hidden form input that s visible in edit form, combining with useRef/forwardRef to  get the value, but will not update using useRef as research showed its not advised to directly manipulate the DOM with useREf

Reply:
I'm not sure what you'd use the hidden input or ref for, my point was more that the data isn't being used - shouldn't it get e.g. POSTed somewhere?